src/hierarchical_deterministic/bip39/bip39_entropy.rs

```diff
@@ -0,0 +1,185 @@
+use crate::{decl_secret_bytes, prelude::*, UniffiCustomTypeConverter};
+
+macro_rules! entropy_with_byte_counts {
```

Comment:
TODO document macro a bit.

Ok.. fasten your seatbealts @micbakos-rdx and @GhenadieVP ... this macro - `entropy_with_byte_counts` calls the other macro `decl_secret_bytes` to declare all the assocated types: `Entropy32Bytes`, `Entropy28Bytes`, `Entropy24Bytes` etc...